roles/migrationcontroller/tasks/main.yml

```diff
@@ -40,6 +40,12 @@
       mig_ui_config_namespace: openshift-config
     when: config_namespace_check.resources|length > 0
 
+  - name: "Set up mig operator namespace"
+    k8s:
+      state: "present"
+      definition: "{{ lookup('template', 'mig_manager_namespace.yml.j2') }}"
+    when: not olm_managed and (migration_ui or migration_controller or migration_velero)
```

Comment:
Is this back because you need to rebase or are we creating the namespace again for some reason? I think @djwhatle just removed all of these.

Reply:
Good catch @jmontleon, my guess is we need to take this out.